lua/pomodoro/ui.lua

```diff
@@ -64,65 +55,74 @@ local function createBufferOpts()
         height = win_height,
         row = row,
         col = col,
-        border = "single",
+        border = "rounded",
     }
     return opts
 end
 
----@class PomodoroUI
```

Comment:
Why removing these comments ?

Reply:
well its basically my first time using lua and play around with nvim plugin :) . so I did not notice about it